internal/ingress/annotations/canary/main.go

```diff
@@ -59,6 +60,11 @@ func (c canary) Parse(ing *networking.Ingress) (interface{}, error) {
 		config.Weight = 0
 	}
 
+	config.WeightTotal, err = parser.GetIntAnnotation("canary-weight-total", ing)
+	if err != nil {
+		config.WeightTotal = 100
+	}
```

Comment:
in Lua script, < 100 value is ignored and the value defaults to 100, should we add this logic here too?